selection/dbms/postgres_dbms.py

```diff
@@ -72,9 +72,19 @@ def create_database(self, database_name):
         self.exec_only("create database {}".format(database_name))
         logging.info("Database {} created".format(database_name))
 
-    def import_data(self, table, path, delimiter="|"):
-        with open(path, "r") as file:
-            self._cursor.copy_from(file, table, sep=delimiter, null="")
+    def import_data(self, table, path, delimiter="|", encoding=None):
+        if encoding:
+            with open(path, "r", encoding=encoding) as file:
```

Comment:
"r" is the default parameter and can be ommitted
encoding=None is the default paramter: we can use the `if` after `open`
https://docs.python.org/3/library/functions.html#open

Reply:
Great, thanks!